lib/spack/spack/solver/asp.py

```diff
@@ -379,7 +392,9 @@ def raise_if_unsat(self):
         constraints = self.abstract_specs
         if len(constraints) == 1:
             constraints = constraints[0]
-        conflicts = self.format_minimal_cores()
+
+        debug = spack.config.get('config:debug', False)
+        conflicts = self.format_cores() if debug else self.format_minimal_cores()
```

Comment:
Any chance that `format_cores()` could eventually call `colify()` to display the facts?